src/main/scala/org/clulab/asist/agents/DacClientBase.scala

```diff
@@ -0,0 +1,160 @@
+package org.clulab.asist.agents
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl._
+import akka.http.scaladsl.model._
+import com.typesafe.scalalogging.LazyLogging
+import org.clulab.asist.messages._
+import org.json4s.{Extraction,_}
+import org.json4s.jackson.JsonMethods._
+import org.json4s.jackson.Serialization
+import org.json4s.jackson.Serialization.{read,write}
+import org.json4s.JField
+
+
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.util.control.NonFatal
+import scala.util.{Failure, Success}
+
+/**
+ * Authors:  Joseph Astier, Adarsh Pyarelal, Rebecca Sharp
+ *
+ * Updated:  2021 July
+ *
+ * Manage communications with the Dialog Act Classification (DAC) server
+ * @param agent Owner class with callback methods
+ */
+class DacClientBase (
```

Comment:
What's the difference between this and the DacClient? they seem to have very similar methods/functionalities?

Reply:
Agreed, the base class should have common methods.

Reply:
does DacClient extend Base? I missed that

Reply:
This file has been removed.